src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -250,6 +253,14 @@ public void datasetTest() throws Exception {
                     region,
                     equalTo(testSettingRegion));
             }
+
+            assertThat("dataset has billing profiles returned from the database",
+                fromDB.getDatasetSummary().getBillingProfiles(),
+                is(not(emptyIterable())));
```

Comment:
Why `emptyIterable()` instead of `empty()`?
```suggestion
                is(not(empty())));
```

Reply:
Because I'm not all that familiar with all the different matchers Hamcrest provides 😅